pex/bin/pex.py

```diff
@@ -653,6 +653,12 @@ def _compatible_with_current_platform(interpreter, platforms):
 
 
 def main(args=None):
```

Comment:
Putting the override here means that the override works for both `python -m pex` and the binary created by `scripts/package.py`.

Reply:
`python -m pex.cli` is already set up for that style access.